gen3-cli/cot/backend/test/templates/cf_structure_obj_block.py

```diff
@@ -125,83 +142,18 @@ def errors(self):
             self.__errors = errors
             return errors
 
-
-def test():
-    body = {
-        'path': {
-            'exists': []
-        },
-        'not': {
-            'empty': {
-                'list': [1, 2],
-                'scalar': 1,
-                'obj': {
-                    '1': 'value',
-                    '2': 'value',
-                    '3': 'value'
-                }
-            }
-        },
-        'empty': {
-            'list': [],
-            'scalar': None,
-            'obj': {}
-        },
-        Template.RESOURCES_KEY: {
-            'TestResource': {
-                Template.RESOURCE_TYPE_KEY: 'TestType'
-            }
-        },
-        Template.OUTPUT_KEY: {
-            'TestOutput': {
-                'Value': 'value'
-            }
-        }
-    }
-    template = Template(body)
-
-    template.match('path.exists', [])
-
-    template.resource('TestResource', 'TestType')
-
-    template.output('TestOutput')
-
-    template.len('not.empty.list', 2)
-    template.len('not.empty.obj', 3)
-
-    template.exists('path.exists')
-
-    template.not_empty('not.empty.list')
-    template.not_empty('not.empty.scalar')
-    template.not_empty('not.empty.obj')
-    assert not template.errors
-
-    template = Template(body)
-    template.match('path.exists', [1])
-    template.match('not.empty.scalar', 10)
-    template.match('not.empty.obj', {'1': 'value'})
-    assert len(template.errors) == 3
-
-    template = Template(body)
-    template.resource('NotExists', 'TestType')
-    template.resource('TestResource', 'WrongType')
-    assert len(template.errors) == 2
-
-    template = Template(body)
-    template.output('NotExists')
-    assert len(template.errors) == 1
-
-    template = Template(body)
-    template.len('not.empty.list', 3)
-    template.len('not.empty.obj', 2)
-    assert len(template.errors) == 2
-
-    template = Template(body)
-    template.exists('not.path.exists')
-    assert len(template.errors) == 1
-
-    template = Template(body)
-    template.not_empty('empty.list')
-    template.not_empty('empty.scalar')
-    template.not_empty('empty.obj')
-    assert len(template.errors) == 3
+    def assert_structure(self):
+        import json
+        if self.errors:
+            raise AssertionError(json.dumps(self.errors, indent=4))
+
+
+def structure_test(filename):
+    try:
+        with open(filename, "rt") as f:
+            import json
```

Comment:
I see no reasons to use local imports here (re and json). Doesn't seem to increase performance in any considerable way, and the readability is decreased.

Reply:
It's a part of the template. Some parts may not be included at all therefore local imports are used.